FhirToDataLake/native/parquet/csharp/src/ParquetException.cs

```diff
@@ -28,13 +28,13 @@ private static string GetParquetErrorMessage(int status)
         {
             switch (status)
             {
-                case 10001:
+                case ParquetConverterErrorCodes.ReadInputJsonError:
                     return "Input json is invalid.";
```

Comment:
define const string for these error message?

Reply:
Updated, thanks.